.github/workflows/run_tests.yaml

```diff
@@ -17,33 +19,35 @@ jobs:
         docker-compose run re_api sh scripts/run_tests.sh
         docker-compose down --remove-orphans
 
-  docker_build_and_push:
-    runs-on: ubuntu-latest
-    needs: run_tests
-    if: (github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/master') && github.event_name == 'push' && !contains(github.event.head_commit.message, 'skip_docker_build')
```

Comment:
This line skips the docker build/push if the commit message contains 'skip_docker_build'. I think it would be more sensible to add documentation about this and have the builds pushed to dockerhub by default.